.contributebot

```diff
@@ -1,3 +1,4 @@
 {
   "issue_title_pattern": "^.*$"
+  "pull_request_title_pattern": "^.*$"
```

Comment:
I'm OK with that, but note that the issue pattern is already clear here, so there is an inconsistency between issues and PRs. If we're fine with that, I can just adjust the message, no problem

Reply:
The issue pattern is there because to end-users there's conceptually one component to report bugs against. For contributors to Wire, there are different components, so I think the PR rule is fine as-is. I would prefer we change the message only.